test/libsolidity/syntaxTests/experimental/parsing_stdlib_import_3.sol

```diff
@@ -3,6 +3,7 @@ pragma experimental solidity;
 import { identity } from std.stub;
 // ====
 // EVMVersion: >=constantinople
+// via-ir: true
```

Comment:
Don't we already have an option for this? I.e. `// compileViaYul: true`?

Reply:
That's in semantic tests. But good point, I guess would be good to have the same name here.

Reply:
That's for semantic tests, this is for syntax. Although having a discrepancy in naming is kinda lame, but I like `via-ir` better.

Reply:
> That's in semantic tests. But good point, I guess would be good to have the same name here.

`compileViaYul` is a string option though, and has "also", "true" and "false" as possible options. What does "also" do?

Reply:
Runs legacy and also IR.

Reply:
Do I just port `compileViaYul` here as is - i.e. as a string option, or will bool suffice, (just true/false)? I prefer `via-ir` though.

Reply:
IMO it would be better not to introduce a new name if both really mean the same thing. But I would not mind a refactor in a separate PR to rename them if you really don't like this name.

As for the value, I'd keep it as a string to have the possibility of introducing `"also"` open but for now I'd only implement `"true"`/`"false"`. Having `"also"` available here (and even as the default) would make sense in the longer term but it's something not worth spending time on right now.

Reply:
I'd honestly prefer a refactor, because renaming now is just gonna make rebasing of newAnalysis (and PRs that branch off of it) even more of a nightmare.

Reply:
What? How? The PR touches only 7 lines. I had zero problems rebasing my PRs on `newAnalysis` after previous, much more extensive changes. I can't see how this can be worse.

Reply:
Ah, lol, forgot I already rebased `newAnalysis`.

Reply:
> That's for semantic tests, this is for syntax. Although having a discrepancy in naming is kinda lame, but I like `via-ir` better.

hum...maybe the `AnalysisFramework` and `ExecutionFramework` should share compiler settings so we didn't need to have such duplication.

Reply:
They definitely should. I actually planned to unify them a bit during my last series of refactors, but I had enough other changes that I decided that it's something for another day :)

Reply:
So what needs to be done in order to merge this in? This PR is really just a cherry-pick of the experimental callgraph PR, so I'd rather not spend time on a refactor here. Also, what about the naming and typing of the flag? `compileViaYul` and string, or leave it as is? I need this for the callgraph PR in order to merge it, so that I can fully focus on the forest of strongly connected components.

Reply:
My opinion is still as stated above, so either:
- rename to `compileViaYul`, make it a string and done.
- Use you name (just tweak to match style), make it a string. Then submit a PR to rename  `compileViaYul` to `viaIR`.

The first one should be quick and simple and that's what I'd do. But if you choose the second, the whole refactor is still just a single rename, so I can't see that being too complex either.